apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -91,7 +90,9 @@ class AnimationPickerListItem extends React.Component {
       }
     ];
     const iconImageSrc = this.props.category
-      ? this.props.categoryImagePathPrefix + `${this.props.category}.png`
+      ? `/blockly/media/gamelab/animation-previews/category_${
```

Comment:
i know this was previously the directory for category previews, but i think it's confusing that both gamelab and spritelab use the `gamelab/` directory. in other places, we've condensed logic like this into a general `p5lab` component/class/whatever (e.g., `<P5LabView/>`) -- should we do the same here? (so the new path would be `/blockly/media/p5lab/animation-previews`)

Reply:
I'm not entirely sure how that /blockly/media path works but it can't find the previews when I move it outside

Reply:
i'm not sure i understand -- when you move the images to `apps/static/p5lab/animation-previews/` you can't see them? they may be cached or somehow initialized on start-up, so you might need to restart the apps server if that's the case

Reply:
<img width="990" alt="Screen Shot 2021-01-20 at 11 03 48 AM" src="https://user-images.githubusercontent.com/17147070/105222299-2ea2f280-5b0f-11eb-8cdc-50b082a220bf.png">
I just see this, it says that there's no image there. I restarted the apps server and the dashboard server a few times, it still won't show up

Reply:
interesting, it still thinks its at gamelab no matter how many times I restart